src/node/in_memory_ext.rs

```diff
@@ -29,18 +29,11 @@ impl<S: ForkSource + std::fmt::Debug + Clone + Send + Sync + 'static> InMemoryNo
     /// # Returns
     /// The applied time delta to `current_timestamp` value for the InMemoryNodeInner.
     pub fn increase_time(&self, time_delta_seconds: u64) -> Result<u64> {
-        self.get_inner()
-            .write()
-            .map_err(|err| anyhow!("failed acquiring lock: {:?}", err))
-            .map(|mut writer| {
-                if time_delta_seconds == 0 {
-                    return time_delta_seconds;
-                }
-
-                let time_delta = time_delta_seconds.saturating_mul(1000);
```

Comment:
Not sure if I am missing something but I don't think there ever was a reason to multiply by a 1000 here

Reply:
I guess the assumption was that `current_timestamp` was in milliseconds.